java/vector/src/main/java/org/apache/arrow/vector/dictionary/Dictionary.java

```diff
@@ -52,14 +52,6 @@ public String toString() {
   }
 
   @Override
-  public boolean equals(Object o) {
```

Comment:
Well, this should probably be looked at as a separate issue anyway and since I'm no longer using it for this PR, I'll revert it.  I created https://issues.apache.org/jira/browse/ARROW-1184 for this.